.github/workflows/multus.yaml

```diff
@@ -111,3 +111,9 @@ jobs:
         # independent from other tests as long as nodes are tainted and labeled
         if: steps.taint.outcome == 'success' && !cancelled()
         run: ./tests/scripts/multus/test-240-stretch-cluster-only.sh
+
+      - name: consider upterm debugging
```

Comment:
I dont see it in the CI test,

We need to make sure if any test fails, we can connect to both tmate and upterm at the same time